jetty-jspc-maven-plugin/pom.xml

```diff
@@ -10,7 +10,6 @@
   <name>Jetty :: Jetty JSPC Maven Plugin</name>
   <properties>
     <bundle-symbolic-name>${project.groupId}.jspc.plugin</bundle-symbolic-name>
-    <it.debug>false</it.debug>
```

Comment:
What is it.debug, and why is it being removed for something to do with the jetty-runner?